Contributing.md

```diff
@@ -0,0 +1,59 @@
+# Contributing to DeliverEase
+
+Welcome to DeliverEase! We are excited to have you contribute to our project. DeliverEase is made using Django (Python) for the backend and HTML/CSS for the frontend. Your contributions can help us improve and enhance the platform, empowering delivery partners and connecting communities even more effectively.
+
+## How to Contribute
+
+To contribute to DeliverEase, please follow these guidelines:
+
+1. Fork the repository to your GitHub account.
+
+2. Create a new branch from the `main` branch. Use a descriptive name for your branch that reflects the nature of your contribution.
+
+3. Make your changes to the codebase or documentation in your branch. Be sure to follow the coding style and conventions used in the project.
+
+4. Write clear and concise commit messages for your changes.
+
+5. Test your changes thoroughly to ensure they work as expected.
+
+6. Update or create new tests where applicable.
+
+7. Ensure that all existing tests pass successfully before submitting your changes.
+
+8. Push your branch to your forked repository.
+
+9. Create a pull request (PR) from your branch to the `main` branch of the main repository.
+
+10. Provide a detailed description of your changes in the pull request, including the problem you are solving, the approach you took, and any relevant information for reviewers.
+
+11. Be responsive to feedback and comments on your pull request. If requested changes are needed, make the necessary adjustments.
+
+12. Once your pull request is approved and meets the project's quality standards, it will be merged into the main repository.
+
+## Code Style and Guidelines
+
+To maintain a consistent codebase, we follow specific code style and guidelines. Please adhere to the following conventions when contributing:
```

Comment:
Attaching a link to the best practices docs officially would make more sense here.
cc: @Devansh-Singh-Parmar

Reply:
Requesting a reference